packages/rs-sdk/tests/fetch/protocol_version_votes.rs

```diff
@@ -0,0 +1,22 @@
+use crate::{common::setup_logs, config::Config};
+use dashcore_rpc::dashcore::{hashes::Hash, ProTxHash};
+use dpp::util::deserializer::ProtocolVersion;
+use rs_sdk::platform::FetchMany;
+
+/// Given some existing identity ID, when I fetch the identity keys, I get some of them indexed by key ID.
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_protocol_version_votes() {
+    setup_logs();
+
+    let cfg = Config::new();
+    let mut sdk = cfg.setup_api().await;
+
+    let query = ProTxHash::from_slice(&[0u8; 32]).expect("zero protxhash");
+    let votings = ProtocolVersion::fetch_many(&mut sdk, query)
+        .await
+        .expect("fetch protocol version votes by node");
```

Comment:
I don't really understand here, how do you get the count of the elements you want to fetch, or how do you get from none? I think this interface is very confusing.

Reply:
You should use `LimitQuery` , I'll add a test

Reply:
You can't get from none, as the .proto requires this field to be filled: 

```
    bytes start_pro_tx_hash = 1;
```

Should we change that?

Reply:
I added extensive docs, and created a new type `MasternodeProtocolVote`.

Also added conveniance method ` MasternodeProtocolVote::fetch_many_with_limit()`

Hope it's more developer-friendly now.

Reply:
bytes start_pro_tx_hash = 1; can be empty, which translates to none for the query.

Reply:
pushed support for None in protxhash